### PR TITLE
Edits to RST files for PyPrimeMesh

### PR DESCRIPTION
**Pull request by PipKat**
In addition to updating the PyAnsys email, a broken link, and adding info on how to change to a different version of the doc, I skimmed through the content to make lots of editorial suggestions to RST files and the PY files for the examples. I see lots of API reference content doesn't met our PyAnsys guidelines--but I will suggest edits to those in a different PR after this one is merged

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +76,6 @@ with prime.launch_prime() as prime_client:
 PyPrimeMesh is licensed under the MIT license.
 
 PyPrimeMesh makes no commercial claim over Ansys whatsoever. This library extends the functionality of
-Ansys Prime Server by adding a Python interface without changing the core behavior or license
-of the original software. The use of Ansys Prime Server requires a legally licensed copy of Ansys
+the Ansys Prime server by adding a Python interface without changing the core behavior or license
+of the original software. The use of the Ansys Prime server requires a legally licensed copy of Ansys
 Workbench.
```
Comment (PipKat) [inline README.md:81]: @waltersma 
Licensing here mentions Ansys Workbench--not the same as in line 59? These discrepancies also exist in the getting_started.index file.

File: README.md [hunk 2]
```diff
@@ -7,57 +7,62 @@
...
 ```bash
 pip install -e .
 ```
 
 ## Dependencies
 
-You must have Ansys 2023 R1 or newer versions installed for Ansys Prime Server (optionally, CAD readers can be configured).  Ansys Prime Server requires a Preppost or CFD Preppost license to run.
+You must have Ansys 2023 R1 or later installed for the Ansys Prime server.
+(Optionally, CAD readers can be configured.) The Ansys Prime server requires
+a Preppost or CFD Preppost license to run.
```
Comment (PipKat) [inline README.md:59]: @waltersma 
I tried to find out the proper spellings for these products but couldn't determine exact matches. Is the first the ```PrepPost`` component that is part of the Ansys Mechanical Enterprise product and the other Ansys Composite PrepPost (ACP)? Note later in the "License and Acknowledgements" section, it says that Ansys Prime server requires a legally licensed copy of Ansys Workbench. These discrepancies also exist in the index.rst file for the Getting started section.

**Comment by ninad-kamat** [comment]
Doe this also fix #420 ?

File: README.md [hunk 3]
```diff
@@ -7,57 +7,62 @@
 
 ## Overview
 
-PyPrimeMesh provides a python client to Ansys Prime Server. Ansys Prime Server delivers core Ansys meshing technology.
+PyPrimeMesh provides a Python client to the Ansys Prime server. The Ansys Prime server
```
Comment (waltersma) [inline README.md:10]: Ansys Prime Server should be capitalised as this is the name.

File: README.md [hunk 4]
```diff
@@ -7,57 +7,62 @@
...
 
 ```bash
 pip install -e .
 ```
 
 ## Dependencies
 
-You must have Ansys 2023 R1 or newer versions installed for Ansys Prime Server (optionally, CAD readers can be configured).  Ansys Prime Server requires a Preppost or CFD Preppost license to run.
+You must have Ansys 2023 R1 or later installed for the Ansys Prime server.
+(Optionally, CAD readers can be configured.) The Ansys Prime server requires
```
Comment (waltersma) [inline README.md:58]: Seems odd to start a sentence with a bracketed comment.

File: README.md [hunk 5]
```diff
@@ -7,57 +7,62 @@
...
+a PrepPost or a Mechanical or Fluids PrepPost (CFD) license to run.
```
Comment (waltersma) [inline README.md:59]:
```suggestion
a Mech PrepPost or CFD PrepPost license to run.
```

File: README.md (hunk 1 above)
Comment (waltersma) [reply README.md:81]: @PipKat we need unified ansys build installed as this is how we distribute the server.  Please suggest a preferred wording.  Intention here is not to state the license used by the server as this is already stated.  Maybe we simply remove the statement regarding it being legally licensed if you feel that is confusing?

**Comment by PipKat** [comment]
@waltersma The only topic now with lowercase "server" is this one--which I think is fine as using Ansys Prime Server here would unnecessarily add to the length of section titles and text. The first sentence in the first section uses the proper name. Is this acceptable to you?

**Comment by PipKat** [comment]
@waltersma I think this one might be ready for you to review, approve, and accept? (Fingers crossed!)

**Pull request merged**